project/SparkBuild.scala

```diff
@@ -1002,10 +1002,9 @@ object KubernetesIntegrationTests {
         if (excludeTags.exists(_.equalsIgnoreCase("r"))) {
           rDockerFile = ""
         }
-        val extraOptions = if (javaImageTag.isDefined) {
-          Seq("-b", s"java_image_tag=$javaImageTag")
```

Comment:
Simply, `javaImageTag.get`?

Reply:
addressed

Reply:
Thank you for your opinion, @beliefer . I asked to minimize this because I want to backport this to all live release branches (including 3.5). We can adjust that style improvement to Apache Spark 4.1, of course, independently.